Makefile

```diff
@@ -0,0 +1,77 @@
+APP = ask_izzy
+REPO = contyard.office.infoxchange.net.au/$(APP)
```

Comment:
That's the internal Docker container yard that Straddle Carrier uses to load containers.